### PR TITLE
feat(api): issues custom attributes

### PR DESCRIPTION
**Pull request by juanfran**

### CODE REVIEW

File: src/app/api/issues/issues-api.module.ts [hunk 1]
```diff
@@ -19,6 +20,7 @@ import { IssuesApiService } from './issues-api.service';
   ],
   providers: [
     IssuesApiService,
+    IssueCustomAttributeApiService,
```
Comment (Xaviju) [inline src/app/api/issues/issues-api.module.ts:23]: Missing here `IssueCustomAttributeValuesApiService`

File: src/app/api/issues/issue-custom-attribute-api.service.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  constructor(private http: HttpClient, private config: ConfigService) { }
+
+  public get base() {
+    return `${this.config.apiUrl}/issue-custom-attributes`;
+  }
+
+  public list(projectId: number) {
+    return this.http.get<IssueCustomAttribute[]>(this.base, {
+      params: {
+        projectId: projectId.toString(),
```
Comment (bameda) [inline src/app/api/issues/issue-custom-attribute-api.service.ts:25]: This should be `project` instead of `projectId` according to the api documentation.

Comment (juanfran) [reply src/app/api/issues/issue-custom-attribute-api.service.ts]: fixed!

File: src/app/api/issues/issues-api.module.ts (hunk 1 above)
Comment (juanfran) [reply src/app/api/issues/issues-api.module.ts:24]: fixed!

**Pull request merged**